pkg/ipam/ipam.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+// ExecAdd invoke the plugin to add container to network
```

Comment:
ExecAdd invokes the given plugin with the given configuration to add a container to a network